src/OSPSuite.Presentation/Presenters/ParameterIdentifications/ParameterIdentificationFeedbackPresenter.cs

```diff
@@ -73,21 +76,18 @@ protected override void SaveFormLayout(Point location, Size size)
 
       public void Handle(ParameterIdentificationStartedEvent eventToHandle)
       {
-         _currentParameterIdentifications.Add(eventToHandle.ParameterIdentification);
-         _parameterIdentification = eventToHandle.ParameterIdentification;
-         _view.Caption = Captions.ParameterIdentification.FeedbackViewFor(_parameterIdentification.Name);
-         showParameterIdentificationFeedback();
+         if (eventToHandle.ParameterIdentification != _parameterIdentificationFeedback.ParameterIdentification)
```

Comment:
Much better. You even have a _parameterIdentification member so this could be
if (!Equals(_parameterIdentification , eventToHandle.ParameterIdentification))
  return;

Same later.  on  public void Handle(ParameterIdentificationTerminatedEvent eventToHandle)